content/influxdb/clustered/write-data/best-practices/data-lifecycle.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Data ingest lifecycle best practices
+description: >
+  Best practices for managing the lifecycle of data ingested into InfluxDB.
+menu:
+  influxdb_clustered:
+    name: Data ingest lifecycle
+    parent: write-best-practices
+weight: 204
+---
+
+Data ingested into InfluxDB must conform to the retention period of the database
+in which it is stored.
+Points with timestamps outside of the retention period are no longer queryable,
+but may still have references maintained in
+[Object storage](/influxdb/clustered/reference/internals/storage-engine/#object-store)
+or the [Catalog](/influxdb/clustered/reference/internals/storage-engine/#catalog),
+resulting in an increase in operational overhead and cost.
+To reduce these factors, it is important to manage the lifecycle of ingested data.
+
+Use the following best practices to manage the lifecycle of data in your
+InfluxDB cluster:
+
+- [Use appropriate retention periods](#use-appropriate-retention-periods)
+- [Tune garbage collection](#tune-garbage-collection)
+
+## Use appropriate retention periods
+
+When [creating or updating a database](/influxdb/clustered/admin/databases/#create-a-database),
+use a retention period that is appropriate for your requirements.
+Storing data longer than is required adds unnecessary operational cost to your
+InfluxDB cluster.
+
+## Tune garbage collection
+
+Once data falls outside of a database's retention period, the garbage collection
+service can remove all artifacts associated with the data from the Catalog and Object store.
+Tune the garbage collector cutoff period to ensure that data is removed in a timely manner.
+
+Use the following environment variables to tune the garbage collector:
+
+- `INFLUXDB_IOX_GC_OBJECTSTORE_CUTOFF`: the age at which Parquet files not
+  referenced in the Catalog become eligible for deletion from Object storage.
+  The default is `30d`.
+- `INFLUXDB_IOX_GC_PARQUETFILE_CUTOFF`: how long to retain rows in the Catalog
+  that reference Parquet files marked for deletion. The default is `30d`.
+
+{{% warn %}}
+**These must NEVER be set to a value less than `3h` (3 hours).**
+
+IOx requires a grace period before files can be removed.
+{{% /warn %}}
```

Comment:
Where this will be/is enforced server-side, I think we can soften the language a bit.

```suggestion
{{% warn %}}
To ensure there is a grace period before files and references are removed, the
minimum garbage collector (GC) object store and Parquet file cutoff time is
three hours (`3h`).
{{% /warn %}}
```